cloud/aws/deploy/website.go

```diff
@@ -397,51 +525,72 @@ func (a *NitricAwsPulumiProvider) deployCloudfrontDistribution(ctx *pulumi.Conte
 		return err
 	}
 
-	// apply invalidation on the distribution when files change
-	pulumi.All(a.Distribution.ID().ToStringOutput(), a.websiteChangedFileOutputs.ToStringArrayOutput()).ApplyT(func(args []interface{}) error {
-		cdnID := args[0].(string)
-		websiteChangedFileKeys := []string{}
-
-		// Filter out empty strings from the array
-		for _, key := range args[1].([]string) {
-			if key != "" {
-				websiteChangedFileKeys = append(websiteChangedFileKeys, key)
-			}
+	if domainName != "" {
+		_, err = route53.NewRecord(ctx, fmt.Sprintf("cdn-alias-record-%s", a.StackId), &route53.RecordArgs{
+			ZoneId: pulumi.String(hostedZoneId),
+			Type:   pulumi.String("A"),
+			Name:   pulumi.String(strings.Split(".", domainName)[0]),
```

Comment:
This looks like it won't work for sub domain names:

- app.test.com
- docs.test.com